pontoon/base/models.py

```diff
@@ -76,6 +76,10 @@ def cldr_plurals_list(self):
         else:
             return map(int, self.cldr_plurals.split(','))
 
+    @property
+    def dir(self):
```

Comment:
It might, but there's also the argument that having locales in the DB allows a particular Pontoon instance to only bother supporting locales it cares about, and lets us add new locales for new teams without a code change, but that's kind've a weak argument.

It's certainly worth trying out depending on how hard of a change it'd be.